src/main/java/enmasse/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -293,22 +293,26 @@ public void handle(ProtonLink<?> link) {
 								Object obj = this.vertx.sharedData().getLocalMap(this.context.getEbName()).remove(deliveryTag);
 								
 								if (obj instanceof KafkaMessage<?, ?>) {
-	
+									// get partition and offset from delivery tag : <partition>_<offset>
+									int underscore = deliveryTag.indexOf("_");
+									int messagePartition = Integer.valueOf(deliveryTag.substring(0, underscore));
+									long messageOffset = Long.valueOf(deliveryTag.substring(underscore + 1));
```

Comment:
`partition` and `offset` are being used in a enclosing scope so it would be a compile time error to declare them.